docs/manualGrading.md

```diff
@@ -70,6 +57,10 @@ Each question will have its score and/or feedback updated and the total assessme
 
 If you prefer to use points rather than a percentage score, rename the `score_perc` column in the CSV file to `points`.
 
+If a feedback column is provided, it will be shown by default to the student above the submission panel, as seen in the image below. This feedback accept Markdown formatting.
```

Comment:
Nit:
```suggestion
If a `feedback` column is provided, it will be shown by default to the student above the submission panel, as seen in the image below. The feedback can utilize Markdown formatting.
```